BaseApiBundle/Tests/Transformer/TransformerTestCase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace OpenOrchestra\BaseApi\Tests\Transformer;
+
+use Phake;
+
+/**
+ * Test TransformerTestCase
+ */
+abstract class TransformerTestCase extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \Phake_IMock
+     */
+    protected $transformerManager;
+
+    /**
+     * @var \Phake_IMock
+     */
+    protected $router;
+
+    /**
+     * @var \Phake_IMock
+     */
+    protected $groupContext;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $this->router = Phake::mock('Symfony\Component\Routing\RouterInterface');
+        $this->groupContext = Phake::mock('OpenOrchestra\BaseApi\Context\GroupContext');
+        $this->transformerManager = Phake::mock('OpenOrchestra\BaseApi\Transformer\TransformerManager');
+
+        Phake::when($this->router)
+            ->generate
```

Comment:
The missing () is for using wildcard parameters as described in the documentation : http://phake.readthedocs.org/en/latest/method-parameter-matchers.html#id1

> If you are familiar with Phake::anyParameters() then you will recognize that the shorthand functionality is really just short hand of Phake::anyParameters(). You can still use Phake::anyParameters() but it will likely be deprecated at some point in the future.

I'll replace by `generate(Phake::anyParameters)` in my next commit.